modules/account-lib/src/coin/baseCoin/blsKeyPair.ts

```diff
@@ -68,12 +82,42 @@ export abstract class BlsKeyPair implements BaseKeyPair {
    * @param {string} prv A hexadecimal public key to validate
    * @returns {boolean} Whether the input is a valid private key or not
    */
-  isValidBLSPrv(prv: string): boolean {
+  public static isValidBLSPrv(prv: string): boolean {
     try {
       BLS.PrivateKey.fromHexString(prv);
       return true;
     } catch (e) {
       return false;
     }
   }
+
+  /**
+   * Whether the input is a valid BLS private key
+   *
+   * @param {string} prv A hexadecimal public key to validate
+   * @returns {boolean} Whether the input is a valid private key or not
+   */
+  public static isValidBLSPrvBytes(prv: Buffer): boolean {
```

Comment:
Why do we need a bytes and hex string version of the same function? Seems one or the other should suffice and callers can serialize into the input format the function takes.